src/log_outgoing_requests/tests/test_logging.py

```diff
@@ -63,12 +61,33 @@ def test_expected_data_is_saved_when_saving_enabled(self, m):
                 self.assertEqual(request_log.req_content_type, "")
                 self.assertEqual(request_log.res_content_type, "")
                 self.assertEqual(request_log.response_ms, 0)
+                self.assertEqual(
+                    request_log.req_headers,
+                    (
+                        "{'User-Agent': 'python-requests/2.26.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive'}"
+                        if method == "HEAD"
+                        else "{'User-Agent': 'python-requests/2.26.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '0'}"
+                    ),
+                )
+                self.assertEqual(request_log.res_headers, "{}")
```

Comment:
The test should include some data and check for every field we store (I miss `res_headers`, also `req_content_type`, `res_content_type` and `params`)